dev_tools/notebooks/isolated_notebook_test.py

```diff
@@ -69,6 +69,8 @@
     # temporary: need to fix QVM metrics and device spec
     'docs/tutorials/google/spin_echoes.ipynb',
     'docs/tutorials/google/visualizing_calibration_metrics.ipynb',
+    # temporary: allow name changes of keyword arguments in quantum engine interfaces to stabilize
+    'docs/simulate/virtual_engine_interface.ipynb',
```

Comment:
This has since been removed. Looks like you may just need to merge from main to get rid of this.